augur/datasources/metrics_status/metrics_status.py

```diff
@@ -19,15 +19,15 @@ def __init__(self):
 
     def determine_frontend_status(self, metric):
         attribute = None
-
+        
         if metric.metric_type == "timeseries":
-            attribute = next((attribute for attribute in self.timeseries if f"/api/unstable/<owner>/<repo>/timeseries/{attribute[2]}" == metric.endpoint), None)
+            attribute = next((attribute for attribute in self.timeseries if "/api/unstable/<owner>/<repo>/timeseries/{}".format(attribute[2]) == metric.endpoint), None)
```

Comment:
I had to change these 3 `if` statements slightly, as removing the `f` disables the necessary string interpolation. The solution, although less readable (IMO), works just fine and is okay by me.